CHANGELOG.md

```diff
@@ -319,6 +314,10 @@ Adding a new version? You'll need three changes:
 
 ### Changed
 
+- The `expressions` router is now the default for DB-less mode in tests and dev configurations.
+  This is not expected to functionally affect routing, but may affect performance for
+  some configurations.
+  [#4934](https://github.com/Kong/kubernetes-ingress-controller/pull/4934)
```

Comment:
```suggestion
```
I'd say just nix it entirely, since there's no controller-level setting and we generally omit test changes from the changelog as not user-facing.

Unfortunately I originally wrote it with the expectation that we _were_ going to update the chart, but that got derailed.